init-tools.sh

```diff
@@ -22,47 +22,59 @@ case $OSName in
 
     Linux)
         OS=Linux
-        __DOTNET_PKG=dotnet-ubuntu-x64
+        source /etc/os-release
+        if [ $ID = "centos" ]; then
+            __DOTNET_PKG=dotnet-centos-x64
+        elif [ $ID = "ubuntu" -o $ID = "debian" ]; then
+            __DOTNET_PKG=dotnet-ubuntu-x64
+        else
+            echo "Unsupported ID '$ID' detected. Downloading ubuntu-x64 tools."
```

Comment:
I'm not sure if 'ID' is understandable. How about 'Unsuported Linux Distribution ...' or something like that?